docs/fsharp/style-guide/formatting.md

```diff
@@ -832,11 +832,27 @@ with
 
 ## Formatting function parameter application
 
-In general, most function parameter application is done on the same line.
+In general, most function parameter application is done on the same line:
 
-If you wish to apply parameters to a function on a new line, indent them by one scope.
+```fsharp
+let x = sprintf "\t%s - %i\n\r" x.IngredientName x.Quantity
+
+let printListWithOffset a list1 =
+    List.iter (fun elem -> printfn $"%d{a + elem}") list1
+```
+
+When pipelines are concerned, the same is typically also true, where a curried function is applied as an argument on the same line:
+
+```
+let printListWithOffsetPiped a list1 =
+    list1
+    |> List.iter (fun elem -> printfn $"%d{a + elem}")
+```
+
+However, you may wish to apply parameters to a function on a new line, as a matter of readability or because the list of arguments or the argument names are too long. In that case, indent with one scope:
```

Comment:
```suggestion
However, you may wish to pass arguments to a function on a new line, as a matter of readability or because the list of arguments or the argument names are too long. In that case, indent with one scope:
```

Again, perhaps this is an F# term that is getting me a bit mixed up. But when I read "apply parameters" I think that I'm building out a function, which doesn't seem to be the case. Instead, it seems as though we're providing arguments to a function - thus, we "provide arguments" or "pass arguments" - not "apply parameters, right?

Reply:
It's technically more correct to call it parameter application, but I think this is fine for an explanation